src/cascadia/WindowsTerminal/AppHost.cpp

```diff
@@ -1556,3 +1575,22 @@ void AppHost::_CloseRequested(const winrt::Windows::Foundation::IInspectable& /*
     const auto pos = _GetWindowLaunchPosition();
     _logic.CloseWindow(pos);
 }
+
+void AppHost::_AppInitializedHandler(const winrt::Windows::Foundation::IInspectable& /*sender*/,
+                                     const winrt::Windows::Foundation::IInspectable& /*arg*/)
+{
+    // GH#11561: We're totally done being initialized. Resize the window to
+    // match the initial settings, and then call ShowWindow to finally make us
+    // visible.
+
+    LaunchMode launchMode{};
+    _initialResizeAndRepositionWindow(_window->GetHandle(), _proposedRect, launchMode);

```

Comment:
Worried a bit about the resize - how often will we start up at the *wrong* size?

Reply:
I never saw it?

Reply:
lmao well this was the issue now 😑